lib/v2/datasource/remote/api/eos_repository.dart

```diff
@@ -32,4 +31,48 @@ abstract class EosRepository {
 
     return transaction;
   }
+
+  Future<Result> updateProfile({
```

Comment:
this should be in the Profile Repository

Reply:
I thought we determined that Eos would be a repo by itself with the methods and calls it has. 
Now that we could call the Eos lib in each repo to access its classes, but how would it handle the common methods (like buildFreeTransaction) if there is no multiple inheritance?